### PR TITLE
Multicluster via gateways example

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: content/docs/examples/multicluster/gateways/index.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+title: Gateway-connected Clusters
```
Comment (geeknoid) [inline content/docs/examples/multicluster/gateways/index.md]: connected -> Connected

File: content/docs/examples/multicluster/gateways/index.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    $ kubectl label --context=$CTX_CLUSTER2 namespace bar istio-injection=enabled
+    $ kubectl apply --context=$CTX_CLUSTER2 -n bar -f @samples/httpbin/httpbin.yaml@
+    $ export GATEWAY_IP_CLUSTER2=$(kubectl get --context=$CTX_CLUSTER2 svc --selector=app=istio-ingressgateway \
+        -n istio-system -o jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+    {{< /text >}}
+
+1. Create a service entry for the `httpbin` service in `cluster1`.
+
+    To allow `sleep` in `cluster1` to access `httpbin` in `cluster2`, we need to create
+    service entry for it. The host name of the service entry should be of the form
```
Comment (geeknoid) [inline content/docs/examples/multicluster/gateways/index.md]: a service entry

File: content/docs/examples/multicluster/gateways/index.md (hunk 1 above)
Comment (frankbu) [reply content/docs/examples/multicluster/gateways/index.md]: Even though it's hyphenated (`Gateway-Connected Clusters`)? Would that apply to [this one](https://preliminary.istio.io/docs/examples/multicluster/split-horizon-eds/) too (`Cluster-Aware`)?

Comment (geeknoid) [reply content/docs/examples/multicluster/gateways/index.md]: I'm not a native English speaker, so this particular grammatical subtlety is beyond my skill set.

There's got to be a capitalization rule for this...

Comment (frankbu) [reply content/docs/examples/multicluster/gateways/index.md]: According to my quick google search, I think the answer is yes, capitalize it.

**Pull request merged**